test/vendor/fullstory-recording.ts

```diff
@@ -0,0 +1,33 @@
+import { MockClass } from '../mock';
+
+export class FullStory extends MockClass {
+  consent(userConsents?: boolean): void {}
+
+  disableConsole(): void {}
+  
+  enableConsole(): void {}
+  
+  event(eventName: string, eventProperties: object): string | null {
```

Comment:
Can we add source as an optional 3rd argument? We'll be using it to track usage.